examples/remote_cluster.py

```diff
@@ -23,7 +23,7 @@ def main():
     # Define the barer token we are going to use to authenticate.
     # See here to create the token:
     # https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/
-    aToken = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    aToken = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
is the token length following standard for bearer token? Should we just use `"<token>"` or `AbCdEf123456` if the example won't work any way?

Reply:
Updated.